src/index.ts

```diff
@@ -1,10 +1,11 @@
 import OAuth from './OAuth';
 import RestClient from './RestClient';
+import * as types from './types';
 
-const BigCommerce = {
-  Auth: OAuth,
-  Rest: RestClient,
-};
+class BigCommerce {
```

Comment:
🍹 This doesn't necessarily need to be a class - when you run `new BigCommerce.Auth`, the `BigCommerce.Auth` part still resolves to just `Auth` whether BigCommerce is a class or an object. I'd suggest talking with the team to see if there's any good reason to switch the object to a class

Reply:
Updated back to obj